### PR TITLE
docs: document grouping chunks with Vite

### PR DESCRIPTION
**Pull request by dpschen**

### CODE REVIEW

File: docs/guide/advanced/lazy-loading.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ When using a bundler like webpack, this will automatically benefit from [code sp
 
 When using Babel, you will need to add the [syntax-dynamic-import](https://babeljs.io/docs/plugins/syntax-dynamic-import/) plugin so that Babel can properly parse the syntax.
 
-## Grouping Components in the Same Chunk
+## Grouping Components in the Same Chunk with Webpack
```
Comment (posva) [inline docs/guide/advanced/lazy-loading.md:43]:
```suggestion
## Grouping Components in the Same Chunk with webpack
```

**Review by posva — CHANGES_REQUESTED** [review]
Let's add a sub section

```md
### With webpack

...

### With Vite

...
```

File: docs/guide/advanced/lazy-loading.md (hunk 1 above)
Comment (dpschen) [reply docs/guide/advanced/lazy-loading.md]: Changed =)

**Review by posva — APPROVED** [review]
thanks!

**Pull request merged**